pipes/config.yaml

```diff
@@ -192,6 +192,27 @@ vphaser_max_bins: 10
 # allele freq > 0.005. If false, no filtering is performed.
 vcf_merge_naive_filter: false
 
+# Preprocessing method for reads for de novo assembly
+assembly_preproc_method: 'subsamp'
+
+# Method of de novo assembly
+assembly_de_novo_method: 'trinity'
+
+# Method of choosing the reference to use for scaffolding
+refchoice_for_scaffold_method: 'common'
```

Comment:
Can we spell out "reference"? 
`reference_choice_for_scaffold_method `
or maybe:
`reference_selection_method_for_scaffolding`